common/reducers/wallet.js

```diff
@@ -25,7 +25,6 @@ export const INITIAL_STATE: State = {
   inst: null,
   balance: null,
   tokens: {},
-  isBroadcasting: false,
```

Comment:
Maybe its better to keep this, and set it to null. Just to keep the state consistent

Reply:
I'm not sure how that's better -- there is no `isBroadcasting` state for a wallet, only for a specific transaction. 

This is just leftover from when I was figuring out the state shape and forgot to remove it.